mirrord/cli/src/extension.rs

```diff
@@ -6,7 +6,6 @@ use mirrord_progress::{Progress, ProgressMode, TaskProgress};
 use crate::{config::ExtensionExecArgs, error::CliError, execution::MirrordExecution, Result};
 
 /// Faciliate the execution of a process using mirrord by an IDE extension
-#[tokio::main(flavor = "current_thread")]
```

Comment:
Why was this necessary?

Reply:
We didn't have an event loop in the caller, and now we have.